src/wallet.md

```diff
@@ -0,0 +1,121 @@
+### solana-wallet CLI
+
+The [solana crate](https://crates.io/crates/solana) is distributed with a command-line interface tool
+
+Usage:
+
+```sh
+$ solana-wallet [common-options] [command] [command-specific options]
+```
+
+`common-options` include:
+* `--fee xyz` - Transaction fee (0 by default)
```

Comment:
Current common-options:
- network
- keypair path
- gossip timeout
- rpc-port
- proxy (to use TLS for rpc)